docs/docs/release-notes/infrahub/release-1_1_0.mdx

```diff
@@ -24,10 +24,79 @@ title: Release 1.1.0
 
 # Release 1.1.0
 
-This release contains both new features and bug-fixes to resolve issues found in Infrahub v1.0.7 and prior.
+This release contains both new features and bug-fixes to resolve issues found in Infrahub v1.0.10 and prior.
 
 ## Main changes
 
+Infrahub 1.1 brings computed attributes, a new (and highly-requested) feature, as well as many behind-the-scenes changes.
+
+See the detailed changelog in the following sections for more information.
+
+### Computed attributes
+
+The computed attributes feature enables the dynamic calculation of attribute values based on user-defined logic.
+This functionality is valuable in a number of use cases, not least of which is programmatically generating interface or circuit names.
+Several users have requested this capability and we are excited to make it available in Infrahub 1.1.
+
+There are two main ways to configure computed attributes, they can be created using either **Jinja2** or **Python**.
+
+For basic use cases, Jinja2-based computed attributes are now available.
+To use this style of computed attribute, the user writes a concise Jinja2 template directly into their Schema definition file.
+Then, any changes to a field referenced in the template will update the resultant computed attribute automatically.
+There are some limitations with Jinja2-based computed attributes however; for example, only direct relationships can be used (a relationship of a relationship is not accessible).
+
+For more complex use cases, with the need for nested relationships or more detailed logic, Python-based computed attributes are also available.
+Python-based computed attributes leverage the exiting Python-based transform feature.
+Users can define a transformation within the schema to apply to a their field, and the attribute is then generated asynchronously by an Infrahub worker.
```

Comment:
```suggestion
Users can define a transformation within the schema to apply to their field, and the attribute is then generated asynchronously by an Infrahub worker.
```

Reply:
I fixed this typo; thank you for catching it! 👍🏻